v22.2/show-backup.md

```diff
@@ -59,11 +60,12 @@ Field | Description
 `backup_type` |  The type of backup: [full](take-full-and-incremental-backups.html#full-backups) or [incremental](take-full-and-incremental-backups.html#incremental-backups).
 `start_time` | The time of the earliest data encapsulated in the backup. Note that this only displays for incremental backups. For a full backup, this is `NULL`.
 `end_time` | The time to which data can be restored. This is equivalent to the [`AS OF SYSTEM TIME`](as-of-system-time.html) of the backup. If the backup was _not_ taken with [revision history](take-backups-with-revision-history-and-restore-from-a-point-in-time.html), the `end_time` is the _only_ time the data can be restored to. If the backup was taken with revision history, the `end_time` is the latest time the data can be restored to.
-`size_bytes` | The size of the backup, in bytes.
+`size_bytes` | The size of the backup objects, in bytes. Note that `size_bytes` indicates the logical size of the table objects, the sum of the size of each key value pair. See `file_bytes` in this table for more detail.
```

Comment:
nit: `the sum of` is meant to clarify `logical size`, but the connection is lost over the comma. Think we just need a `which` or similar. What about something like:

> The size of the backed-up objects, in bytes. Note that `size_bytes` indicates the logical size of table objects, which is computed as the sum of the size of each key value pair. See `file_bytes` in this table for more detail.